python/GangaPanda/Lib/Panda/Panda.py

```diff
@@ -134,6 +148,16 @@ def convertDQ2NamesToQueueName(locations):
 
 def convertQueueNameToDQ2Names(queue):
 
+    ret=[pandaresources[x]['ddm'] for x in pandaresources if pandaresources[x]['name']==queue]
+    # logger.warning('(%s) ret:%s:' % (type(ret), ret))
+    try:
+        ret=ret[0]
+    except:
+        # logger.warning('oh well')
+        import traceback
+        logger.error('queue %s error %s'%(queue, str(traceback.print_exc() )))
+    return str(ret).split(',')
+
```

Comment:
Something around `convertQueueNameToDQ2Names` stopped working, that was the main reason for the intervention and patch. In HC we are very actively moving far far away from getting topology & status information from combination of sources (PanDA client vs. ToAcache vs. AGIS) and are taking this info from AGIS directly. For that we have collector that produces the json  `+pandaresources_json = '/tmp/agis_pandaresources.json' `. If Ganga can move in the similar direction, everybody wins.